src/main/Controller/SvgFileCreation.java

```diff
@@ -0,0 +1,8 @@
+package Controller;
+
+public interface SvgFileCreation {
+	public void createEmptyFile();
+	public void display();
+	public void modifyFile();
+	public void deleteFile();
```

Comment:
for all of them, the return can be a boolean, because I suppose their will be some try catch inside the methods that can fail.
The methods will return true if the action as been well done.